gulpfile.js

```diff
@@ -21,15 +21,40 @@ gulp.task('default', function() {
     .pipe(concat('vaadin-icons.html'))
     .pipe(modify({
       fileModifier: function(file, contents) {
+        /* eslint-disable max-len */
         // Enclose all icons in an iron-iconset-svg
-        return `<link rel="import" href="../iron-icon/iron-icon.html">
+        return /* html */`<!-- NOTICE: Generated with 'gulp icons' -->
+<!--
+@license
+Copyright (c) 2015-2017 Vaadin Ltd.
+This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+-->
+
+<link rel="import" href="../iron-icon/iron-icon.html">
 <link rel="import" href="../iron-iconset-svg/iron-iconset-svg.html">
-<iron-iconset-svg name="vaadin-icons" size="16">
+
+<!--
+\`vaadin-icons\` is a set of 600+ icons which can be used together with Polymer's [\`iron-icon\`](https://elements.polymer-project.org/elements/iron-icon) component.
+
+To use the \`vaadin-icons\` iconset, import the specific \`vaadin-icons.html\`, and
```

Comment:
Same comment goes to the npm version, so I would recommend to update it, remove the and point to the "installation" docs in README: https://github.com/vaadin/vaadin-icons#installation

Reply:
Updated the text. Is it ok now?